src/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -338,6 +351,29 @@ public GenotypeLikelihoodsAllelePair(final int alleleIndex1, final int alleleInd
         return cache;
     }
 
+
+    /**
+     * Calculate the cache of anyploid alleles for each PL index
+     * Creates the ordering for all possible combinations of ploidy alleles. Computed recursively and the
+     * result is stored in a cache.
+     *
+     * @param numAlleles    Number of alleles (including ref)
+     * @param ploidy        Number of chromosomes in set
+     * @param genotype      An entry of ploidy alleles
+     */
+    protected static void calculatePLcache(final int numAlleles, final int ploidy, final List<Integer> genotype){
```

Comment:
and rename this one calculateAnyPloidyPLcache (BTW, the thing you are refering to is spelled 'aneuploid' or 'aneuploidy' I'm specifically using the spelling above to indicate 'any ploidy'...i.e. not only diploid.